Auto find cur

lbliii opened the pull request:
- updated onboarding pg 
- updated CUR pg 
- updated IAM page ([minimal permissions section)](https://docs.metricly.com/integrations/aws-integration/aws-iam-installation/#alternative-create-a-custom-policy-with-minimal-permissions-read-only-role) by swapping out the old policy  with the more limited s3 read policy. 
- inter-linked these sections to point to each other 

I wasn't able to see the slack link in the ticket for creating a new section, but happy to get that added in today as well.

TheConnMan commented on content/integrations/AWS-Integration/AWS-IAM-Installation.md:
```diff
@@ -125,58 +125,40 @@ If you want to use a limited read only access policy, you’ll need to create a
-    "Resource": "*"
-  }
-]
+  "Version": "2012-10-17",
```
Wasn't this policy for monitoring?

lbliii replied:
I'm unsure. That policy also contains s3 permissions, but more broadly right?

Should i keep the old section and clarify them as two separate minimum perm policies? one based in cost and the other monitoring?

TheConnMan replied:
I'm not sure either to be honest. @rclagett what do you think?

rclagett replied:
It is for monitoring, which we still do. So the policy should be kept. If we want a third policy  - minimum for cost performance  metrics that would be EC2, EBS, RDS, S3, ELB, ASG - then we could do that.

The three would be minimal for monitoring, minimal for cost performance metrics, minimal for billing files.

rclagett commented on content/getting-started/onboarding-wizard.md:
```diff
@@ -76,16 +71,15 @@ It can take up to a few hours for data to populate in the S3 bucket.
-   - **Detailed Billing Source**: select **Cost and Usage Report (recommended)** for your Detailed Billing Source. Detailed Billing Files are a legacy feature of AWS and may not be supported in the future.
-   - **S3 Bucket Name**: Select the s3 bucket you created in [section 1, part 2](/getting-started/onboarding-wizardv2/#2-enable-cost-usage-reports-in-aws).
-   - **Report Path Prefix**: Input the path prefix of an existing CUR or the new one created in section 1, part 2. `CostAndUsageReports/HourlyCSVWithResourceIDs`
+1. Leave **Cost Explorer API** enabled.
```
```suggestion
1. Leave **Enable Cost Explorer** enabled.
```

rclagett commented on content/getting-started/onboarding-wizard.md:
```diff
@@ -76,16 +71,15 @@ It can take up to a few hours for data to populate in the S3 bucket.
-   - **S3 Bucket Name**: Select the s3 bucket you created in [section 1, part 2](/getting-started/onboarding-wizardv2/#2-enable-cost-usage-reports-in-aws).
-   - **Report Path Prefix**: Input the path prefix of an existing CUR or the new one created in section 1, part 2. `CostAndUsageReports/HourlyCSVWithResourceIDs`
+1. Leave **Cost Explorer API** enabled.
+2. Select a **Report Path Prefix**. This path should be connected to the S3 bucket where Cost and Usage Report files are stored.
```
```suggestion
2. Select a **Report Path Prefix**. The values listed are a combination of the S3 bucket name where the billing files reside followed by the full Report path prefix of the Cost & Usage report you configured.
```

rclagett replied:
If we don't have access to the CUR endpoint then this selection doesn't work and it reverts back to the old manual way. Should we keep the old instructions that were removed in case we don't have that access? I'm afraid a user may get confused with no other recourse in the docs if they git presented the two new fields

rclagett commented on content/integrations/AWS-Integration/aws-cur.md:
```diff
@@ -63,8 +66,11 @@ The following best practices ensure optimal Cost & Usage report setup.
 ### Request CUR Setup for Resold Resources
 
-You may not have access to your billing preferences if you have purchased AWS services through a reseller. You must contact your reseller and request to be set up with an S3 bucket to store your CUR data.
+You may not have access to your billing preferences if you have purchased AWS services through a reseller. You must contact your reseller and request to be set up with an S3 bucket to store your CUR data. You can then use [this minimal read-only IAM role](http://localhost:1313/integrations/aws-integration/aws-iam-installation/#alternative-create-a-custom-policy-with-minimal-permissions-read-only- ...
```
```suggestion
You may not have access to your billing preferences if you have purchased AWS services through a reseller. You must contact your reseller and request to be set up with an S3 bucket to store your CUR data. This [minimal read-only IAM role](http://localhost:1313/integrations/aws-integration/aws-iam-installation/#alternative-create-a-custom-policy-with-minimal-permissions-read-only-role) can be used to allow CloudWisdom read-only access to a single S3 bucket containing the billing files that help to power our cost reporting features.
```

rclagett commented on content/integrations/AWS-Integration/aws-cur.md:
```diff
@@ -36,25 +36,28 @@ Already have an existing Cost and Usage Report with GZIP or ZIP compression set
+![cur-lookup](/images/aws-cur/cur-lookup.png)
 5. Select **Save**.  
 
+{{% notice tip %}}
```
I do like this tip a lot. Could we still have the old setup steps just in case we don't have the CUR access?

rclagett commented on content/integrations/AWS-Integration/aws-cur.md:
```diff
@@ -36,25 +36,28 @@ Already have an existing Cost and Usage Report with GZIP or ZIP compression set
-4. Scroll to the **Detailed Billing Settings** section and complete the following fields:
-   - **S3 Bucket Name**: the case sensitive name of bucket chosen in section 1, step 7.
-   - **Report Path Prefix**: Input the path prefix of an existing CUR or the new one created in section 1, step 8: `CostAndUsageReports/HourlyCSVWithResourceIDs` 
+3. Ensure both **Cost Explorer API** and **Detailed Billing** are enabled.
```
Cost Explorer isn't part of CUR and the toggle name just changed to "Cost & Usage Reports"
```suggestion
3. Ensure **Cost & Usage Reports** is enabled.
```

rclagett commented on content/integrations/AWS-Integration/aws-cur.md:
```diff
@@ -36,25 +36,28 @@ Already have an existing Cost and Usage Report with GZIP or ZIP compression set
-   - **S3 Bucket Name**: the case sensitive name of bucket chosen in section 1, step 7.
-   - **Report Path Prefix**: Input the path prefix of an existing CUR or the new one created in section 1, step 8: `CostAndUsageReports/HourlyCSVWithResourceIDs` 
+3. Ensure both **Cost Explorer API** and **Detailed Billing** are enabled.
+4. Scroll to the **Detailed Billing Settings** section.
```
Renamed to "Cost & Usage Reports"
```suggestion
4. Scroll to the **Cost & Usage Reports settings** section.
```

rclagett commented on content/integrations/AWS-Integration/AWS-IAM-Installation.md:
```diff
@@ -96,24 +97,47 @@ If you already have an existing IAM role for CloudWisdom but it does not include
+### Minimal Monitoring Permissions
 
-#### Alternative: Create a Custom Policy with Minimal Permissions (Read Only Role)
+Grants read-only access to collect CloudWatch **performance metrics** for all AWS services and their integrations.
```
```suggestion
Grants read-only access to collect CloudWatch **performance metrics** for the AWS services CloudWisdom provides an integration.
```

lbliii commented:
updated the json, swapped minimal billing to cost, and added the blurbs. 

also added a tip for minimal cost permissions related to connecting a restricted integration to the master billing account using these perms for a dedicated IAM role. (let me know if i misunderstood how that works)

rclagett commented on content/integrations/AWS-Integration/AWS-IAM-Installation.md:
```diff
@@ -96,24 +97,47 @@ If you already have an existing IAM role for CloudWisdom but it does not include
+There are four options available for this role: standard permissions, minimal monitoring permissions, minimal cost permissions, and master billing account permissions. A Minimal permission policy must be created _before_ being assigned to an IAM Role.
+
+{{% notice tip %}}
+Virtana recommends opening a second browser tab to follow this section when creating minimal read-only policies. Once the policy is created, use the original tab to resume setup of the IAM role.
```
```suggestion
We recommend opening a second browser tab to follow this section when creating minimal read-only policies. Once the policy is created, use the original tab to resume setup of the IAM role.
```

rclagett commented on content/integrations/AWS-Integration/aws-cur.md:
```diff
@@ -36,35 +36,39 @@ Already have an existing Cost and Usage Report with GZIP or ZIP compression set
+Some customers store their billing files in a shared master billing account and need to grant CloudWisdom restricted access to one specific S3 bucket. You can use the [master billing account permissions](/integrations/aws-integration/aws-iam-installation/#master-billing-account-permissions) to create a limited read-only IAM role when configuring your AWS integration with CloudWisdom.
 
-You may not have access to your billing preferences if you have purchased AWS services through a reseller. You must contact your reseller and request to be set up with an S3 bucket to store your CUR data.
+For customers using a shared S3 bucket: Virtana only reads the costs for accounts that we monitor; we discard all data for any unrelated accounts.
```
```suggestion
For customers using a shared S3 bucket: CloudWisdom only reads the costs for accounts that we monitor; data for unrelated accounts is discarded.
```
